pyrealm/pmodel/two_leaf_irradience.py

```diff
@@ -0,0 +1,31 @@
+"""To do."""
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+class TwoLeafIrradience:
+    """Calculates two-leaf irradience."""
+
+    def __init__(self, beta_angle: NDArray, PPFD: NDArray, LAI: NDArray, PATM: NDArray):
+
+        self.beta_angle: NDArray = beta_angle
+
+
+def beta_angle(lat: NDArray, d: NDArray, h: NDArray) -> NDArray:
+    """Calculates solar beta angle.
+
+    Calculates solar beta angle using Eq A13 of dePury & Farquhar (1997).
+
+    Args:
+        lat: array of latitudes (rads)
+        d: array of declinations (rads)
+        h: array of hour angle (rads)
```

Comment:
I've adopted Keith's short variable names in my rough and ready translation to Python, but we should shift to longer names (`latitude`, `declination`, `hour_angle`)?